src/init/features/dataconnect/index.spec.ts

```diff
@@ -33,6 +36,7 @@ describe("init dataconnect", () => {
       expectedSource: string;
       expectedFiles: string[];
       expectCSQLProvisioning: boolean;
+      expectEnsureSchemaGQL: boolean;
```

Comment:
No change required, but this could've been optional, and you wouldn't have to change any of the existing test-cases